index.js

```diff
@@ -2,17 +2,37 @@
 //.
 //. [Fluture][] type definitions for [Sanctuary][].
 //.
+//. ## Usage
+//.
 //. ```console
-//. $ npm install --save fluture sanctuary-def fluture-sanctuary-types
+//. $ npm install --save fluture-sanctuary-types
 //. ```
 //.
-//. ## Usage
+//. Note that you also need [Fluture][] and [sanctuary-def][] installed.
+//. Sanctuary-def comes preinstalled with Sanctuary, so you could install
+//. either one. Fluture has to be installed separately. See `package.json`
+//. for compatible versions (defined in `peerDependencies`).
+//.
+//. ### EcmaScript Module
 //.
 //. ```js
 //. import $ from 'sanctuary-def';
-//. import {env} from 'fluture-sanctuary-types';
+//. import {env, FutureType} from 'fluture-sanctuary-types/index.js';
+//. import {resolve} from 'fluture/index.js';
```

Comment:
Why do you prefer this approach to importing `Future` wholesale?

Reply:
I actually prefer `import * as fl from 'fluture/index.js'`, but for code example I like to used named imports to illustrate that there's named exports.